code/modules/projectiles/gun_system.dm

```diff
@@ -652,6 +655,30 @@ User can be passed as null, (a gun reloading itself for instance), so we need to
 		SEND_SIGNAL(gun_user, COMSIG_MOB_ATTACHMENT_FIRED, target, src, master_gun)
 	gun_user?.client?.mouse_pointer_icon = 'icons/effects/supplypod_target.dmi'
 
+/obj/item/weapon/gun/proc/set_shoot_inactive_hand(mob/user, firing) // Handles akimbo
+	if(!user)
+		return
+	if(!dual_wield)
+		user.shoot_inactive_hand = FALSE
+		return
+	
+	var/obj/item/weapon/gun/active_gun = user.get_active_held_item()
+	var/obj/item/weapon/gun/inactive_gun = user.get_inactive_held_item()
+	if(firing) // Apply akimbo delay
```

Comment:
this doesn't need to be in this proc(from l667 to l671). Put that l844

Reply:
Done